src/main/java/school/lemon/changerequest/java/Matrix.java

```diff
@@ -0,0 +1,126 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 11.12.16.
+ */
+public class Matrix {
+
+    private int rows;
+    private int columns;
+    private double[][] matrix;
+
+
+    public Matrix(int rows, int columns) {
+        if ((rows < 1) || (columns < 1)) {
+            System.out.println("Cannot create a matrix smaller that 1x1");
+            System.exit(0);
```

Comment:
Throw appropriate exception instead of System.exit. Also never use such kind of call, cause it could lead to some unpredictable behavior.